ember-flight-icons/addon/components/flight-icon.hbs

```diff
@@ -10,5 +12,12 @@
   viewBox="0 0 {{this.size}} {{this.size}}"
   xmlns="http://www.w3.org/2000/svg"
 >
-  <use href='{{this.contextRootURL}}@hashicorp/ember-flight-icons/icons/sprite.svg#{{@name}}-{{this.size}}'></use>
+  {{#if @title}}
+    <title id={{this.titleId}}>{{@title}}</title>
+    <g role="presentation" class="flight-icon-g">
+      <use href='{{this.contextRootURL}}@hashicorp/ember-flight-icons/icons/sprite.svg#{{@name}}-{{this.size}}'></use>
+    </g>
+  {{else}}
+    <use href='{{this.contextRootURL}}@hashicorp/ember-flight-icons/icons/sprite.svg#{{@name}}-{{this.size}}'></use>
+  {{/if}}
```

Comment:
praise: nice! an elegant piece of code (this whole bloc) :)

Reply:
Thank you ^_^